### PR TITLE
Create User testing doc

### PR DESCRIPTION
**Pull request by kookster**
Describe the user testing, and some of our best practices around it

### CODE REVIEW

File: team/User-testing.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  * Even if the tester is in person, use zoom/slack/hangouts so remote team members can join and observe
+  * Include info about screen sharing for the participant
+* Also schedule a debrief meeting after the testing session to discuss findings with team members
+  * Ideally, immediately following testing, or ASAP
+
+## Preparation
+* Make sure any login credentials are ready, and tested to make sure the tester can get into the application
+* Check any data is setup and makes sense, ideally use data the tester will find meaningful
+* Have team member roles assigned
+* Make sure the A/V setup is working
```
Comment (sandikbarr) [inline team/User-testing.md:31]: We have also been preparing the questions and user activities/scenarios in advance

Comment (kookster) [reply team/User-testing.md:31]: oh, good one! I knew I was forgetting important things

**Comment by kookster** [comment]
thanks for the review - any other changes @sandikbarr or is this good to go?

**Comment by pkarman** [comment]
@sandikbarr @kookster can this be merged and closed?

**Review by sandikbarr — APPROVED** [review]

**Pull request merged**